redaxo/src/core/lib/sql/sql.php

```diff
@@ -738,12 +738,24 @@ public function getValue($column)
      * @param string $column Name of the column
      *
      * @throws rex_sql_exception
-     *
-     * @return array
      */
-    public function getArrayValue($column)
+    public function getArrayValue($column): array
```

Comment:
Der native Return-Type ist ein BC-Break für Leute, die die Klasse erweitern.
Eher unwahrscheinlich, dass die Methode überschrieben wird, aber ich würde es trotzdem eher beim PHPDoc belassen (in R5).